### ISSUE TITLE
'alias' is now a mandatory option?

### ISSUE THREAD

**Issue by AndrewEdmonds11** [alcap-org/AlcapDAQ#210]
I've just had a few errors of the form:

```
terminate called after throwing an instance of 'modules::missing_option'
  what():  Missing option for 'PlotTAP_Integral': 'alias'
```

Does this mean 'alias' is now a compulsory option? Is it necessary since the production config file as some modules defined without one?

**Comment by benkrikler** [comment]
That's a mistake on my part that I overlooked this.  I'll push a quick fix but in the meantime giving an alias should resolve this?

**Comment by benkrikler** [comment]
This should be solved in commit 51b4f56d12ac1beee5bfe8fc26023ef3e93c9453.  I've also made sure to catch these exceptions in the correct place in b8d2099.  I'm closing this, but feel to re-open if there're any other bugs.
